examples/vpc-with-multiple-az/variables.tf

```diff
@@ -23,3 +23,9 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "ami_id" {
+  description = "AMI id in case we want to deploy from AMI"
```

Comment:
But we always deploy from an AMI.
Remove the default value, it should come from the main module.